pkg/kubelet/kubelet_getters.go

```diff
@@ -235,6 +237,15 @@ func (kl *Kubelet) GetNode() (*v1.Node, error) {
 	if kl.kubeClient == nil {
 		return kl.initialNode(context.TODO())
 	}
+	// if we have a valid kube client, we wait up to 5s for initial lister to sync
+	if !kl.nodeHasSynced() {
+		err := wait.PollImmediate(time.Second, 5*time.Second, func() (bool, error) {
```

Comment:
choose a weird number so we can find this later if it comes up. How about 8 seconds